src/query_parser/v2/parser.y

```diff
@@ -446,20 +446,26 @@ expr(A) ::= modifier(B) COLON text_expr(C) . {
 expr(A) ::= modifierlist(B) COLON text_expr(C) . {
 
     if (C == NULL) {
+        for (size_t i = 0; i < Vector_Size(B); i++) {
+          char *s;
+          Vector_Get(B, i, &s);
```

Comment:
`Vector_Get` Makes a copy of an element to the pointer. This does produce a double-free warning.
`Vector_Free` is sufficient.

Reply:
The element was a pointer to a string that should be freed!